### ISSUE TITLE
Not sure how to build the project

### ISSUE THREAD

**Issue by JulianG** [BrettJephson/ashjs-asteroids-example#1]
Hi Brett,

I must be doing something wrong. Is this how you build your project?

I change dir to /build/ and run grunt.cmd (windows) and I get a lot of errors, starting with:

```
Running "lint:files" (lint) task
Linting ../src/asteroids-app.js...ERROR
[L1:C8] Unexpected space after '('.
define( 'asteroids-app', ["brejep/fillsnfixes", "game/asteroids"],
[L1:C28] Mixed double and single quotes.
define( 'asteroids-app', ["brejep/fillsnfixes", "game/asteroids"],
[L1:C50] Mixed double and single quotes.
define( 'asteroids-app', ["brejep/fillsnfixes", "game/asteroids"],
[L2:C13] Missing space after 'function'.

(2000 lines like those, then)

[L32:C31] Too many statements per function (14).
        api.update = function( time ) {
[L3:C14] Too many statements per function (11).
    function( System, RenderNode ) {
>> Too many errors. (5% scanned).

Linting ../src/game/systems/systempriorities.js...ERROR
[L1:C8] Unexpected space after '('.
define( "game/systems/systempriorities",
[L9:C2] Missing semicolon.
)
[L1:C1] 'define' is not defined.
define( "game/systems/systempriorities",

<WARN> Task "lint:files" failed. Use --force to continue. </WARN>

Aborted due to warnings.
```

Thanks in advance!

**Comment by BrettJephson** [comment]
I'll have a look into this. I think the grunt build file needs tidying up. 

The command will be <code>grunt.cmd compile</code> but I think there is an issue with the node modules that means these might need removing from the repository. I'll tidy this up and add a package.json file to let each user install dependencies. It will probably be some time later today.

**Comment by JulianG** [comment]
Thanks! I'll check again later today or tomorrow.
BTW, I saw your .gitignore and it contains a *.fdproj. Are you using FlashDevelop for this?

**Comment by BrettJephson** [comment]
I'm a big fan of FlashDevelop and I did use it for a bit for my Javascript projects. However, I've now moved to using the JetBrains WebStorm IDE. 

I still use FlashDevelop for Actionscript and Haxe projects but, for Javascript dev, I haven't found a better IDE than WebStorm. Unfortunately, it isn't an open source project.

**Comment by BrettJephson** [comment]
Hi, it has been a busy week so I've only now found time to fix the build issues. 

I've simplified and updated the build scripts. If you have node and npm installed, the following steps should work:

```
npm install
grunt
```

The first command installs all the dependencies defined in package.json and the second command will run jshint and requirejs. The output is asteroids-game.js in the build folder and can be played through asteroids.html.
